starter-code/src/viking.js

```diff
@@ -1,11 +1,54 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+
+  receiveDamage(damage) {
+    this.health = this.health - damage;
+  }
+}
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+
+  receiveDamage(damage) {
+    super.receiveDamage(damage);
```

Comment:
Substituir por `this.health = this.health - damage;`

O `super` deve ser chamado dentro de construtores. Quando for sobreescrever um método da classe pai, é necessário reescrever todo o novo método.